recipes/onnx/all/patches/cmake-absl-proto-targets.patch

```diff
@@ -0,0 +1,26 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -220,8 +220,8 @@ if(NOT ONNX_BUILD_CUSTOM_PROTOBUF)
+         set(Build_Protobuf OFF)
+         if("${Protobuf_VERSION}" VERSION_GREATER_EQUAL "4.22.0")
+           # There are extra dependencies for protobuf.
+-          find_package(absl REQUIRED)
+-          find_package(utf8_range)
++          find_package(absl REQUIRED CONFIG)
```

Comment:
Is `CMAKE_FIND_PACKAGE_PREFER_CONFIG` not enough? The `CMakeToolchain` applies it automatically; usually we do not need to enforce `CONFIG` in the `find_package`